spark/src/main/scala/geotrellis/spark/io/accumulo/AccumuloLayerUpdater.scala

```diff
@@ -32,7 +32,7 @@ class AccumuloLayerUpdater[K: Boundable: JsonFormat: ClassTag, V: ClassTag, M: J
       case e: UnsupportedOperationException => throw new LayerUpdateError(id, ": empty rdd update").initCause(e)
```

Comment:
This is in reference to getKeyBounds comment above. I wasn't going to change this because I expect that function to be removed soon. But I think you're right. This site should match and the `getKeyBounds` should return Bounds